<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="TimeSeriesTreeView" name="mTimeSeriesTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsFilterLineEdit" name="tbFilterExpression">
        <property name="placeholderText">
         <string>Show all dates &amp; sources</string>
        </property>
        <property name="qgisRelation" stdset="0">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnUseRegex">
        <property name="text">
         <string>...</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnCaseSensitive">
        <property name="text">
         <string>...</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <widget class="QToolBar" name="mToolBar">
   <property name="windowTitle">
    <string>Time Series</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="optionFollowCurrentDate"/>
  </widget>
  <action name="optionFollowCurrentDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="eotsv_resources.qrc">
     <normaloff>:/eotimeseriesviewer/icons/mapview.svg</normaloff>:/eotimeseriesviewer/icons/mapview.svg</iconset>
   </property>
   <property name="text">
    <string>Follow Current Date</string>
   </property>
   <property name="toolTip">
    <string>Activate to focus the view on the current date</string>
   </property>
  </action>
  <action name="optionUseRegex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>.*</string>
   </property>
   <property name="toolTip">
    <string>Activate to filter by regular expression</string>
   </property>
  </action>
  <action name="optionCaseSensitive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aa</string>
   </property>
   <property name="toolTip">
    <string>Activate to filter case sensitive</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TimeSeriesTreeView</class>
   <extends>QTreeView</extends>
   <header>eotimeseriesviewer.timeseries.widgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="eotsv_resources.qrc"/>
  <include location="../../../QGIS/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
